tutor/src/api/index.js

```diff
@@ -0,0 +1,505 @@
+      trigger: 'loadUpdates',
+      onSuccess: 'loadedUpdates',
+      url: 'notifications',
+      handledErrors: ['*']
+    });
+    connectModelUpdate(User.constructor, 'saveTourView', {
+      pattern: 'user/tours/{id}'
+    });
+    connectModelRead(UserTerms, 'fetch', {
+      onSuccess: 'onLoaded',
+      url: 'terms'
+    });
+    connectModelUpdate(UserTerms, 'sign', {
+      onSuccess: 'onSigned',
+      pattern: 'terms/{ids}',
+      method: 'PUT'
+    });
+    connectModelRead(Purchases.constructor, 'fetch', {
+      onSuccess: 'onLoaded',
+      url: 'purchases'
+    });
+    connectModelUpdate(Purchase, 'refund', {
+      onSuccess: 'onRefunded',
+      pattern: 'purchases/{item_uuid}/refund',
+      method: 'PUT',
+      data: function() {
+        return {
+          survey: this.refund_survey
+        };
+      }
+    });
+    connectModelCreate(User.constructor, 'logEvent', {
+      pattern: 'log/event/{category}/{code}',
+      data: function(arg) {
+        var data;
+        data = arg.data;
+        return {
+          data: data
+        };
+      }
+    });
+    connectModelRead(Offerings.constructor, 'fetch', {
+      url: 'offerings',
+      onSuccess: 'onLoaded'
+    });
+    connectModelCreate(CourseCreate, 'save', {
+      onSuccess: 'onCreated'
+    });
+    connectModelRead(TeacherTaskPlans.constructor, 'fetch', {
+      pattern: 'courses/{courseId}/dashboard',
+      onSuccess: 'onLoaded',
+      params: function(arg) {
+        var endAt, id, startAt;
+        id = arg.id, startAt = arg.startAt, endAt = arg.endAt;
+        return {
+          start_at: startAt,
+          end_at: endAt
+        };
+      }
+    });
+    connectModelUpdate(Student, 'save', {
+      pattern: 'user/courses/{courseId}/student',
+      onSuccess: 'onSaved'
+    });
+    connectModelCreate(CourseEnroll, 'create', {
+      url: 'enrollment_changes',
+      onSuccess: 'onApiRequestComplete',
+      onFail: 'setApiErrors'
+    });
+    connectModelUpdate(CourseEnroll, 'confirm', {
+      pattern: 'enrollment_changes/{id}/approve',
+      method: 'PUT',
+      onSuccess: 'onApiRequestComplete',
+      onFail: 'setApiErrors'
+    });
+    connectModelRead(CourseStudentTasks, 'fetch', {
+      onSuccess: 'onLoaded',
+      pattern: 'courses/{courseId}/dashboard'
+    });
+    return connectModelDelete(StudentTask, 'hide', {
+      onSuccess: 'onHidden',
+      pattern: 'tasks/{id}'
+    });
+  };
+
+  BOOTSTRAPED_MODELS = {
+    user: User,
+    courses: Courses,
+    payments: Payments,
+    hypothesis: Hypothesis
+  };
+
+  start = function(bootstrapData) {
+    var data, model, storeId;
+    for (storeId in BOOTSTRAPED_MODELS) {
+      model = BOOTSTRAPED_MODELS[storeId];
+      data = bootstrapData[storeId];
+      model.bootstrap(data);
+    }
+    return startAPI();
+  };
+
+  module.exports = {
+    startAPI: startAPI,
+    start: start
+  };
+
+}).call(this);
+
+//# sourceMappingURL=index.js.map
```

Comment:
I think this and the index.js.map got checked in by default 😁

Reply:
Also the flux/course.js below

Reply:
I removed them and pushed the changes.